unittests/testattachments.py

```diff
@@ -195,14 +195,14 @@ def filedata(chunk_size=4096):  # pylint: disable=unused-argument
                 attachment = self.arch.attachments.download(
                     IDENTITY,
                     fd,
-                    query={"allow_insecure": True, "allow_not_scanned": True},
+                    query={"allow_insecure": True, "strict": True},
                 )
                 args, kwargs = mock_get.call_args
                 self.assertEqual(
                     args,
                     (
                         f"url/{ROOT}/{ATTACHMENTS_SUBPATH}/{IDENTITY}"
-                        "?allow_insecure=True&allow_not_scanned=True",
+                        "?allow_insecure=True&strict=True",
```

Comment:
I think this is now a valid request - since `strict=True` is optional but when specified `allow_insecure=True` is required to download attachments with scanned_status != SCANNED_OK

Reply:
this is a unittest - only testing the paramaters get passed so dont care here.

However the functests are a different storya sthey actually connect - see functets/execattachments.py